utils/error_injection.hh

```diff
@@ -235,8 +256,8 @@ public:
         return true;
     }
 
-    void enable(const std::string_view& injection_name, bool one_shot = false) {
-        _enabled.emplace(injection_name, injection_data{one_shot});
+    void enable(const std::string_view& injection_name, bool one_shot = false, error_injection_parameters parameters = {}) {
```

Comment:
jfyi params is a common abbreviation for parameters, but it's a matter of taste

Reply:
If both versions are acceptable, I will stay with `parameters`.